serialization/decorator.py

```diff
@@ -0,0 +1,63 @@
+from flask_app import api
+from flask import request, make_response, Response
+from functools import wraps
+from pydantic import BaseModel
+
+
+def _make_json_response(
+    content: BaseModel,
+    status_code: int,
+    by_alias: bool,
+    exclude_none: bool = False,
+    many: bool = False,
+) -> Response:
+    """serializes model, creates JSON response with given status code"""
+    json_content = content.json(exclude_none=exclude_none, by_alias=by_alias)
+
+    response = make_response(json_content, status_code)
+    response.mimetype = "application/json"
+    return response
+
+
+def serialize(
+    query=None,
+    headers=None,
+    cookies=None,
+    resp=None,
+    tags=(),
+    before=None,
+    after=None,
+    response_by_alias: bool = False,
+    exclude_none: bool = False,
+    on_success_status: int = 200,
+):
+    def decorate_validation(func):
```

Comment:
```suggestion
    def decorate_validation(route):
```

... mais c'est du chipotage (et peut-être une question de goût). :)